### PR TITLE
Currency handling while donating.

### PR DESCRIPTION
**Pull request by ostaptan**
Resolves [github issue](https://github.com/pivorakmeetup/pivorak-web-app/issues/498)

### Description

Gets currency from liqpay merchant and stores it in Donations table. (Added currency column for admin panel)

### How to test instructions
 
Run `rspec spec/services/liqpay/charge/handler_spec.rb`

### Pre-merge checklist
 
- [ ] The PR relates to a single subject with a clear title and description
- [ ] Verify that feature branch is up-to-date with `development` (if not - rebase it). 

### Screenshots

| Before                                      | After                                       |
| ------------------------------------------- | ------------------------------------------- |
| Insert screenshots and/or screen recordings | Insert screenshots and/or screen recordings |

### CODE REVIEW

**Comment by ostaptan** [comment]
This needs more testing I guess. But it is not so obvious to find test suits. Needs more investigation of liqpay merchant.

**Review by sveredyuk — DISMISSED** [review]
Well done

File: app/services/donation/create.rb [hunk 1]
```diff
@@ -30,7 +30,8 @@ def create_donation
     def donation_params
       {
         user_id: user_id,
-        amount:  data['amount']
+        amount:  data['amount'],
```
Comment (kushniryb) [inline app/services/donation/create.rb]: Table alignment would look good here.

File: spec/factories/liqpay_factory.rb [hunk 2]
```diff
@@ -2,6 +2,8 @@
   factory :liqpay_valid_response_data, class: Hash do
     status     :success
     amount     Faker::Number.number(3)
+    currency   Faker::Lorem.word.first(3).upcase
+    # currency   Faker::Currency.code why isnt this working?
```
Comment (kushniryb) [inline spec/factories/liqpay_factory.rb]: Let's remove commented code.

File: spec/services/liqpay/charge/handler_spec.rb [hunk 3]
```diff
@@ -38,6 +38,20 @@
...
     end
 
+    context 'currency saved' do
+      let(:response_with_customer) do
+        Base64.strict_encode64(
+          (build :liqpay_valid_response_data, :customer, payment_id: 999).to_json
+        )
+      end
+
+      let(:currency) { (build :liqpay_valid_response_data)[:currency] }
```
Comment (kushniryb) [inline spec/services/liqpay/charge/handler_spec.rb]: `build(:liqpay_valid_response_data)[:currency]` ?
Formatting looks weird.

**Review by kushniryb — CHANGES_REQUESTED** [review]
A few small changes would be appreciated 🐧

File: app/services/donation/create.rb (hunk 1 above)
Comment (ostaptan) [reply app/services/donation/create.rb]: done

**Review by kushniryb — DISMISSED** [review]
🐧

**Review by sveredyuk — APPROVED** [review]

**Review by sveredyuk — DISMISSED** [review]

**Comment by denys-medynskyi** [comment]
Liqpay is replaced by wayforpay payments 😢 
Also we have some legal issues with donations.

**Pull request closed without merging**